intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -814,26 +814,27 @@ \subsection{Generate your own e-mail notifications}
 
 You can instruct the \hpc to send an e-mail to your e-mail address whenever a
 job \strong{b}egins, \strong{e}nds and/or \strong{a}borts, by adding the
-following lines to the job script ``fibo.pbs'':
+following lines to the job script \lstinline|fibo.pbs|:
 
 \begin{code}{bash}
 #PBS -m b
 #PBS -m e
 #PBS -m a
-#PBS -M <your e-mail address>
+#PBS -M john.smith@example.com
```

Comment:
This is actually not need, by default the system will mail to the university email address to which your VSC account is coupled, so just remove the `-M` option?